core/parachain/validator/impl/parachain_processor.cpp

```diff
@@ -1346,19 +1357,53 @@ namespace kagome::parachain {
     auto se = pm_->getStreamEngine();
     BOOST_ASSERT(se);
 
+    // TODO(turuslan): #1757, LargeStatement
+    auto message = std::make_shared<
+        network::WireMessage<network::ValidatorProtocolMessage>>(
+        network::ValidatorProtocolMessage{
+            network::StatementDistributionMessage{network::Seconded{
+                .relay_parent = relay_parent, .statement = statement}}});
+
+    std::unordered_set<network::PeerId> group_set;
+    if (auto r = runtime_info_->get_session_info(relay_parent)) {
+      auto &[session, info] = r.value();
+      if (info.our_group) {
+        for (auto &i : session.validator_groups[*info.our_group]) {
```

Comment:
our_group out of bound of validator_groups?

Reply:
No, `RuntimeInfo::get_session_info` computes `our_group` from `validator_groups`, so it's bound